src/components/slices/intro-description.js

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react'
+import { graphql } from 'gatsby'
+import { GatsbyImage, StaticImage } from 'gatsby-plugin-image'
+import { PrismicRichText } from '@prismicio/react'
+import { Container, Button } from '../Components'
+import * as sty from './intro-description.module.scss'
+import { BiArrowBack } from 'react-icons/bi'
+
+export const IntroDescription = ({ slice }) => {
+  return (
+    <div
+      className={`${sty.introBox} ${
+        slice.primary.palm_displayed ? sty.palm : ''
+      }`}
+    >
+      <div className={sty.textBox}>
+        <div className={sty.titleText}>
+          <PrismicRichText
+            field={slice.primary.intro_description_title.richText}
+          />
+        </div>
+
+        <div className={sty.bodyText}>
+          <PrismicRichText
+            field={slice.primary.intro_description_text.richText}
+          />
+        </div>
+      </div>
+
+      <div className={sty.buttonArea}>
+        <button className={sty.viewWorkButton}>
+          <span className={sty.btnContent}>
+            <PrismicRichText
+              field={slice.primary.intro_description_btn_label.richText}
+            />
+            <BiArrowBack
+              size={18}
+              style={{
+                transform: 'scaleX(-1)',
+                marginLeft: '0.5em',
+              }}
+            />
+          </span>
+        </button>
+      </div>
+    </div>
+  )
+}
+
+export const query = graphql`
+  fragment PageDataBodyIntroDescription on PrismicPageDataBodyIntroDescription {
+    id
+    primary {
+      intro_description_title {
+        richText
```

Comment:
For this title I would use the title field instead because it really should be an h2 minimum since it's high on the page, and also the text needs to be semi-bold which will be annoying to do when it's wrapped in a p tag.